### ISSUE TITLE
using deprecated filter for admin columns line 419. 

### ISSUE THREAD

**Issue by PodsBot** [pods-framework/pods#4101]
using deprecated filter for admin columns line 419.  submitted via Slack by jimtrue

Reported by Steve West in Slack Chat:

I don’t know where this should go but I’m running Pods 2.6.8 and just had an update for Admin Columns Pro to v4.0. I am getting Function Deprecated errors in the ACP Dashboard Help tab:
`The filter cac/column/meta/value used on this website is deprecated since 3.0. The callback used is PodsMeta::cpac_meta_value` with a link to their documentation here: https://www.admincolumns.com/documentation/filter-reference/cac-column-meta-value/
Their documentation for the updated filter is at: https://www.admincolumns.com/documentation/filter-reference/ac-column-value/

I found the deprecated filter (cac/column/meta/value)  on line 419 of /pods/classes/Pods.Meta.php
```    
public function integrations () {
    // Codepress Admin Columns 2.x
    add_filter( 'cac/storage_model/meta_keys', array( $this, 'cpac_meta_keys' ), 10, 2 );
    add_filter( 'cac/post_types', array( $this, 'cpac_post_types' ), 10, 1 );
    add_filter( 'cac/column/meta/value', array( $this, 'cpac_meta_value' ), 10, 3 );
}
```
hope this helps.

**Comment by jimtrue** [comment]
@tobiasschutter wanted you to be aware of this one, too. this is probably on our side and we could do a quick PR to fix it.

**Comment by jimtrue** [comment]
The error/warning is only in the Admin Screens under Help:

![screenshot 2017-04-04 17 14 55](https://cloud.githubusercontent.com/assets/3052875/24678877/5643749a-195a-11e7-8bf3-f374c8fba15d.png)

**Comment by davidmabelle** [comment]
Still getting this error even though this issue is showing as fixed.

The filter cac/column/meta/value used on this website is deprecated since 3.0. The callback used is PodsMeta::cpac_meta_value. View documentation »

**Comment by pglewis** [comment]
Do you have a development site where you're able to test the 2.x branch (not yet released, will become 2.7.10)?  We had another PR for Admin Columns deprecation in there (#5129), so it may have been resolved there.  

If it's still occurring on 2.x I think we should open a new fresh ticket rather than glomming onto history here.
